### DISCUSSION TITLE
GitHub "git clone" times out when cloning from git:// protocol

### DISCUSSION THREAD

**Discussion by GustNash** [community/community#23103] [New to GitHub]
<p>This just popped up for us today on one older Rails app that has some dependencies that link to a <code>git://github.com</code> repo address. It was functioning as of a week or so ago but just stopped today during a routine build. I verified it myself by manually trying to clone a public repo:</p>
<p><code>git clone git://github.com/thoughtbot/griddler.git</code></p>
<p>That hangs for a couple minutes then times out. But if you clone the <code>https</code> protocol version of that repo it works fine.</p>
<p>I couldn’t find any documentation or news about sunsetting the git protocol or other forum posts.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>There’s this changelog entry, which links an announcement last year:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://github.blog/changelog/2022-03-15-removed-unencrypted-git-protocol-and-certain-ssh-keys/">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/5814919/181074870-2f34a040-aaed-407c-99be-907a65a55593.png" class="site-icon" width="32" height="32">

      <a href="https://github.blog/changelog/2022-03-15-removed-unencrypted-git-protocol-and-certain-ssh-keys/" target="_blank" rel="noopener">The GitHub Blog</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/362;"><img src="https://user-images.githubusercontent.com/5814919/181074871-0eada83c-9570-4b63-a775-1027fff7ddad.png" class="thumbnail" width="690" height="362" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/9/2905067aa69fcaa104e46bd00f643595126a02bf_2_690x362.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/9/2905067aa69fcaa104e46bd00f643595126a02bf_2_1035x543.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/2/9/2905067aa69fcaa104e46bd00f643595126a02bf.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/9/2905067aa69fcaa104e46bd00f643595126a02bf_2_10x10.png"></div>

<h3><a href="https://github.blog/changelog/2022-03-15-removed-unencrypted-git-protocol-and-certain-ssh-keys/" target="_blank" rel="noopener">Removed unencrypted Git protocol and certain SSH keys | GitHub Changelog</a></h3>

  <p>Removed unencrypted Git protocol and certain SSH keys</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Reply by ifsdk** [reply]
what a stupid monkeys decision!

**Comment by GustNash** [comment]
<p>Thank you so much! That kinda certainty was exactly what I was looking for before going to town on our Gemfiles.</p>
